packages/console/src/pages/JwtClaims/MonacoCodeEditor/config.ts

```diff
@@ -0,0 +1,24 @@
+import { type EditorProps } from '@monaco-editor/react';
+
+import type { IStandaloneThemeData } from './type';
+
+// Logto dark theme extends vs-dark theme
+export const logtoDarkTheme: IStandaloneThemeData = {
+  base: 'vs-dark',
+  inherit: true,
+  rules: [],
+  colors: {
+    'editor.background': '#181133', // :token/code/code-bg
+  },
+};
+
+// @see {@link https://microsoft.github.io/monaco-editor/typedoc/interfaces/editor.IStandaloneEditorConstructionOptions.html}
+export const defaultOptions: EditorProps['options'] = {
+  minimap: {
+    enabled: false,
+  },
+  wordWrap: 'on',
```

Comment:
nit: Maybe 'off', and scroll with horizontal scrollbar?

Reply:
Based on the figma design.  Sounds like a product decision.

Reply:
ok